test/Microsoft.Framework.PackageManager.FunctionalTests/DnuInstallTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Framework.CommonTestUtils;
+using Microsoft.Framework.PackageManager.FunctionalTests;
+using NuGet;
+using Xunit;
+
+namespace Microsoft.Framework.PackageManager
+{
+    [Collection(nameof(PackageManagerFunctionalTestCollection))]
+    public class DnuInstallTests
+    {
+        private readonly PackageManagerFunctionalTestFixture _fixture;
+
+        public DnuInstallTests(PackageManagerFunctionalTestFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        public static IEnumerable<object[]> RuntimeComponents
+        {
+            get
+            {
+                return TestUtils.GetRuntimeComponentsCombinations();
+            }
+        }
+
+        [Theory]
+        [MemberData(nameof(RuntimeComponents))]
+        public void DnuInstall_WithProjectPathArgument(string flavor, string os, string architecture)
+        {
+            var runtimeHomePath = _fixture.GetRuntimeHomeDir(flavor, os, architecture);
+            using (var tempDir = new DisposableDir())
+            {
+                var packagesDir = Path.Combine(tempDir, "packages");
+                var projectDir = Path.Combine(tempDir, "project");
+                Directory.CreateDirectory(projectDir);
+                var projectJsonPath = Path.Combine(projectDir, Runtime.Project.ProjectFileName);
+                File.WriteAllText(projectJsonPath, @"{
+  ""dependencies"": { }
+}");
+
+                VerifyDnuInstall(
+                    runtimeHomePath,
+                    packageName: "alpha",
```

Comment:
Ah!!!! This is the package that ends up in my packages folder. I think there is a test that leaves that package on the disk. I always have it the global packages folder